backend/msgpack/writer_test.go

```diff
@@ -22,7 +22,11 @@ package msgpack
 
 import (
```

Comment:
Might be better to move the benchmark tests to its own file? `writer_benchmark_test.go`?